js/libs/keycloak-js/dist/keycloak.d.ts

```diff
@@ -248,7 +255,14 @@ export interface KeycloakLoginOptions {
 	/**
 	 * Sets the `acr` claim of the ID token sent inside the `claims` parameter. See section 5.5.1 of the OIDC 1.0 specification.
 	 */
-	acr?: Acr;
+	acr?: Acr
```

Comment:
Looks like a semicolon was removed here.
```suggestion
	acr?: Acr;
```